noetic/distribution.yaml

```diff
@@ -2458,6 +2458,17 @@ repositories:
       url: https://github.com/ros-visualization/gl_dependency.git
       version: kinetic-devel
     status: maintained
+  gmcl:
+    doc:
+      type: git
+      url: https://github.com/adler-1994/gmcl.git
+      version: noetic-devel
```

Comment:
So now this line needs updating. Just like the `source` entry, the version here in the `doc` entry needs to match the name of a branch that is in your repository.